dockerfiles/run/guice/cassandra/Dockerfile

```diff
@@ -28,4 +28,4 @@ ADD destination/conf /root/conf
 VOLUME /logs
 VOLUME /root/conf
 
-ENTRYPOINT java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ -jar james-server.jar 
+ENTRYPOINT java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ "${JVM_OPTIONS}" -jar james-server.jar
```

Comment:
why it's the only place you wrap the string with quotes ?

Reply:
Because I need some coffee.